Digizuite/Extensions/MetadataValueServiceExtensions.cs

```diff
@@ -24,13 +25,13 @@ public static class MetadataValueServiceExtensions
         {
             var response = await service.GetRawMetadata(request, cancellationToken, accessKey);
 
-            var labelValues = response.Values
-                .GroupBy(v => v.LabelId)
+            var itemValues = response.Values
+                .GroupBy(v => v.ItemId)
                 .ToDictionary(g => g.Key, g => g.ToList());
 
             return response.Fields.SelectMany(field =>
             {
-                if (!labelValues.TryGetValue(field.LabelId, out var values))
+                if (!itemValues.TryGetValue(field.ItemId, out var values))
```

Comment:
This is absolutely not correct. We need the label id separation to be able to handle multiple languages.

Reply:
Yeah I was too fast here